tests/phpunit/integration/Core/Admin/ScreensTest.php

```diff
@@ -99,30 +99,73 @@ public function test_removal_of_admin_notices_outside_sitekit( $hookname ) {
 		$this->assertNotEmpty( ob_get_clean() );
 	}
 
-	public function test_menu_order() {
-		$menu_order = array(
-			'index.php',
-			'third-party-plugin',
-			'edit.php',
-			'options-general.php',
-			'googlesitekit-dashboard',
+
+	public function menu_order_provider() {
```

Comment:
Nit-picking, but WP core (and also most of Site Kit) always names the data provider methods `data_*` similarly to the `test_*` method name (i.e. in this case `data_menu_order`).

Reply:
I noticed that too (although they have many providers that don't use this convention as well), and we have both in our codebase (I'm pretty sure I've always used `_provider` as a suffix) but I'm happy to standardize on something.

Reply:
Looked into it, we're at **31 dataProviders**. **17** with **_provider**, **14** with **data_** and this.
```
[/google-site-kit/tests/phpunit/integration]$ ack dataProvider|grep -v vendor|wc -l
      31
[/google-site-kit/tests/phpunit/integration]$ $ ack dataProvider|grep -v vendor|grep _provider |wc -l
      17
[/google-site-kit/tests/phpunit/integration]$ $ ack dataProvider|grep -v vendor|grep data_ |wc -l
      14
```

Reply:
Updated to `data_menu_order`.